x-pack/plugins/monitoring/server/lib/details/get_series.ts

```diff
@@ -22,28 +22,10 @@ import {
 import { formatUTCTimestampForTimezone } from '../format_timezone';
 import { getNewIndexPatterns } from '../cluster/get_index_patterns';
 import { Globals } from '../../static_globals';
+import { Metric } from '../metrics/metrics';
```

Comment:
We can use `import type` here. Doesn't affect the bundle size since it's on the server but it makes it clearer that it's a type that's being imported.